conf.d/source.kubernetes.conf

```diff
@@ -14,6 +15,7 @@
   source_category salt
   source_name k8s_salt
   source_category_prefix "#{ENV['SOURCE_CATEGORY_PREFIX']}"
+	exclude_namespace_regex "#{ENV['EXCLUDE_NAMESPACE_REGEX']}"
```

Comment:
I'm not sure what to do about this. Locally, it shows my indent correctly. On github, it doesn't. :/
<img width="657" alt="screen shot 2017-04-06 at 2 48 03 pm" src="https://cloud.githubusercontent.com/assets/477982/24770489/19cb8ffa-1ad8-11e7-94cd-1c5afc68ba3e.png">